### PR TITLE
Round 2 edits

### PR DESCRIPTION
**Pull request by gtw2**
Using this as a place to compile edits.
- [x] add table of all S&O and parameters
- [x] grammar fixes (still a few)
- [x] remove algorithms
- [x] Need to move process detail
- [x] add approx. same amount of detail to all processes (volox. may need more?) 
- [x] citation fixes
- [x] explain table
- [x] smaller flowsheets for the 4 subprocesses

### CODE REVIEW

**Comment by gtw2** [comment]
Should I cite papers inside the table of parameters?

**Comment by katyhuff** [comment]
> Should I cite papers inside the table of parameters?

Yes, if the parameter value came from somewhere you should cite that. A common way is to have a table structure where one of the columns is "Ref." and contains references where the parameters were found.

**Comment by katyhuff** [comment]
@gtw2 This branch has conflicts. Please resolve them.

**Comment by katyhuff** [comment]
Do not version this file: reduction.synctex.gz

File: winning.tex [hunk 1]
```diff
@@ -0,0 +1,42 @@
+% Author: Kathryn Huff
```
Comment (katyhuff) [inline winning.tex]: I am not the author of this file. Please amend. Say "Authors: Kathryn Huff, Gregory Westphal" or similar.

File: volox.tex [hunk 2]
```diff
@@ -0,0 +1,42 @@
+% Author: Kathryn Huff
```
Comment (katyhuff) [inline volox.tex]: I am not the author of this file. Please amend. Say "Authors: Kathryn Huff, Gregory Westphal" or similar.

**Comment by katyhuff** [comment]
Generally, versioning pdfs is not advised.

File: 2018-westphal-antpc.tex [hunk 3]
```diff
@@ -1,11 +1,11 @@
 \documentclass{anstrans}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\title{PyRe: A Cyclus Pyroprocessing Facility Archetype}
-\author{\textbf{Gregory T. Westphal} and Kathryn D. Huff}
+\title{Pyre: A Cyclus Pyroprocessing Facility Archetype}
+\author{Gregory T. Westphal* and Kathryn D. Huff}
```
Comment (katyhuff) [inline 2018-westphal-antpc.tex]: $^*$

File: 2018-westphal-antpc.tex [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
 \institute{
 Dept. of Nuclear, Plasma and Radiological Engineering, University of Illinois at Urbana-Champaign \\
-gtw2@illinois.edu
+*gtw2@illinois.edu
```
Comment (katyhuff) [inline 2018-westphal-antpc.tex]: $^*$

File: 2018-westphal-antpc.tex [hunk 5]
```diff
@@ -73,159 +77,175 @@ \section{Background: Pyroprocessing}
...
+	\caption{A material balance over the electrowinning sub-process, including observables.}
+	\label{fig:winning}
+\end{figure}
+
+Figure \ref{fig:winning} shows that electrowinning produces Cs waste similar to electroreduction except in reduced quantities. Any fission products remaining in the salt from electrorefining are also removed here. Outside of physically tracked quantities, facility power can also be monitored to observe the use of current to increase throughput. 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Method: \Cyclus Simulation}
-The separations facility provided by the \Cycamore library expansion is used as an initial model of a simple PRIDE facility. 
-The separations archetype allows for the declaration of a feed stream and requires the user-definition of facility efficiencies. 
+The separations facility provided by the \Cycamore library is used as an initial model of a simple PRIDE facility.
```
Comment (katyhuff) [inline 2018-westphal-antpc.tex]: The pride facility has not been explained and KAERI is no longer mentioned or cited (those parts were deleted?). This is not good.

**Review by katyhuff — CHANGES_REQUESTED** [review]

**Comment by katyhuff** [comment]
please see #15 for a pr that fixes some of these issues.

**Comment by katyhuff** [comment]
If you can fix the inline comments in the above review, this can be pulled. Once this is merged, then please move on to #17 .

**Comment by katyhuff** [comment]
bitmoji

**Pull request merged**